### PR TITLE
CDS 814xx release notes

### PR DESCRIPTION
**Pull request by deepakbolangady**
First draft (baseline) for 814xx release notes.

# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the CODEOWNERS. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore
- [x] Release notes

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes.md [hunk 1]
```diff
@@ -16,7 +16,33 @@ For Harness on-prem releases, go to [Harness Self-Managed Enterprise Edition Rel
...
+
+This release does not include new features or enhancements.
+
+#### Early access features
+
+This release does not include early access features.
+
+#### Fixed issues
+
+- The Instance Sync V1 CG did not did not update instance count for Helm Pods when they are removed from the users environment. (CDS-82385, ZD-52612)
```
Comment (brian-f-harness) [inline docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes.md:32]: @deepakbolangady When copying this content to the delegate release notes, I noticed some typos in this RN. I'm also not sure we need to include the info about how to reproduce the issue with FF settings. Here's my suggested content. LMK what you think.

- The instance sync didn't update the instance count for Helm pods when they were removed from the user's environment. (CDS-82385, ZD-52612)

  When synchronizing a native Helm deployment, the assigned `CONTAINER_VALIDATION` tasks returned `null`. As a result, the delegate couldn't pick up this task, and the count didn't update.

  This issue has been resolved. The instance sync will not display the actual instance count immediately after redeploying. The count might take about 10 minutes to update.

**Comment by aimurphy** [comment]
@deepakbolangady Looks like something got messed up? Not sure that these commits should all be here as if they are new commits.

**Comment by deepakbolangady** [comment]
> @deepakbolangady Looks like something got messed up? Not sure that these commits should all be here as if they are new commits.

Yes. I'm killing this PR.

**Comment by deepakbolangady** [comment]
Something got messed up. Killing this PR.

**Pull request closed without merging**